llvm/test/Object/objdump-section-content.test

```diff
@@ -39,3 +39,13 @@ Sections:
 
 # BSS: Contents of section .bss:
 # BSS-NEXT: <skipping contents of bss section at [12c8, 12cc)>
+
+# RUN: yaml2obj %p/Inputs/COFF/mips.yaml | llvm-objdump -s - | FileCheck %s --check-prefix=COFF-MIPS
```

Comment:
What's special about the MIPS behaviour here that requires an additional test of dumping section contents?

Reply:
Nothing special here, I was only adding the test here because other architectures had it.
I'm removing this test.